data/ka'het missions.txt

```diff
@@ -211,8 +211,12 @@ mission "First Contact: Ka'het: Quarg 2"
 			choice
 				`	"What happened to these Builders after that?"`
 			label ka'het
-			`	The Quarg stops for a few seconds before continuing, as if he was reflecting on something. "For many millennia they grew, developing brilliant pieces of technology. Then, compelled by their past experiences, they began creating weapons not only to defend themselves, but also to retaliate; these 'Ka'het', as they named them, defied shortly after they were complete." After saluting you, the Quarg slowly walks away, leaving you with more questions than answers.`
+			`	The Quarg stops for a few seconds before continuing, as if he was reflecting on something. "For many millennia they grew, developing brilliant pieces of technology. Then, compelled by their past experiences, they began creating weapons not only to defend themselves, but also to retaliate; these 'Ka'het', as they named them, defied shortly after they were complete.`
+			`"For few more centuries the Builders lasted, before falling under their very own hubris. Their space is still populated by the Ka'het their gave birth to through the immense facilities they assembled, who do not yet possess a vast intelligence." After saluting you, the Quarg quickly walks away, leaving you with more questions than answers.`
 	on decline
-		log "Asked the Quarg about the history of the Graveyard. Although I wasn't told the reason for the abandoned ringworld, I have learned about the previous inhabitants of the region, the Builders."
+		log "Asked the Quarg about the history of the Graveyard. Although I wasn't told the exact reason for the abandoned ringworld, I have learned about the previous inhabitants of the region, the Builders."
+		log "Factions" "Builders" `The Builders inhabited the Graveyard thousands of years ago, but have fallen extinct long ago. They developed some sort of weapon, called Ka'het, which stopped obeying their orders and insurged against them.`
+		log "Factions" "Ka'het"  `According to the Quarg, the Ka'het were initially a weapon created by the Builders, which then defied and led them to extinction. Even if millennia passed, they are still being produced in the Graveyard.`
```

Comment:
log "Factions" "Ka'het"  `According to the Quarg, the Ka'het were initially a weapon created by the Builders, which then rebelled and led them to extinction. Even if millennia passed, they are still being produced in the Graveyard.`

Reply:
What's the point in changing defied with rebelled?

Reply:
"Defied" does sound strange to me. You're using this in a similar way to the sentence "The Ka'het defied," to which I ask "defied what?" "Defied" usually doesn't just sit alone like that.

Reply:
Still have a strange use of "defied" here.